examples/with_winit/src/lib.rs

```diff
@@ -249,6 +255,8 @@ fn run(
             let width = render_state.surface.config.width;
             let height = render_state.surface.config.height;
             let device_handle = &render_cx.devices[render_state.surface.dev_id];
+            let snapshot = stats.snapshot();
+            let _stats_frame = stats.frame_scope();
```

Comment:
This is unfortunate, because (at least on my machine), this seems to be vsync locked. On the Tiger demo, I get frame times of nearly exactly `16.62ms`, i.e. slightly less than one vsync interval (other comments cover the gap)

Reply:
I agree. In my local benchmarks we were vsync limited (I've been running mostly on 120Hz/90Hz) displays so I don't always hit the limit but vsync does impact the measurement methodology.

Is it possible to configure the winit event loop to not be locked by vsync? Maybe we should use a different (perhaps headless) environment for benchmarks although I really like the interactive set up (I have already identify several adversarial cases by playing with the scene transform alone).

Reply:
I know it's possible - we need to set the wgpu present_mode to *something*

I'd rather stay vsyncy by default through. Maybe another toggle?

Reply:
Done.